### PR TITLE
Header headline section

### PR DESCRIPTION
**Pull request by M0rrighan**
Milestone 1: setup and mobile version skeleton
Added linter, webhint and stylelint configuration files;
Added background images and social networks icons
Added index.html page, with header and heading section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          me.
+        </p>
+        <h3>Let's connect</h3>
+        <div class="social">
+          <a href="#"><img src="icons/twitter.svg" alt="twitter logo" /></a>
+          <a href="#"><img src="icons/linkedin.svg" alt="linkedin logo" /></a>
+          <a href="#"><img src="icons/medium.svg" alt="medium logo" /></a>
+          <a href="#"><img src="icons/github.svg" alt="github logo" /></a>
+          <a href="#"><img src="icons/angellist.svg" alt="angellist logo" /></a>
+        </div>
```
Comment (Davidosky007) [inline index.html]: I suggest you make the list of social icons use a `ul` tag to group them, and `li ` tag per icon; each icon has a link (a tag)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          <h2>Glad to see you!</h2>
+        </hgroup>
+
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h3>Let's connect</h3>
```
Comment (Davidosky007) [inline index.html]: I suggest you make this a button with the necessary styling (hover, active, pressed ) as this is a call to action read this [Link](https://www.smashingmagazine.com/2009/10/call-to-action-buttons-examples-and-best-practices/) for more info

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @M0rrighan, 

Good job so far Love the way you used proper HTML semantics tags 👍 !
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest you start your commit messages with capital letters as this is good practice read [This](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) for further info
# _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (henatan99) [reply index.html]: This issue is addressed well! :clap:

**Review by henatan99 — APPROVED** [review]
Hi @M0rrighan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**